pkg/actions/cluster/cluster.go

```diff
@@ -14,8 +15,14 @@ type Cluster interface {
 }
 
 func New(cfg *api.ClusterConfig, ctl *eks.ClusterProvider) (Cluster, error) {
-	if err := ctl.RefreshClusterStatus(cfg); err != nil {
-		return nil, err
+	err := ctl.RefreshClusterStatus(cfg)
+
+	if err != nil  {
```

Comment:
Did I miss it, is `if err := .. ; err ...` considered harmful?

Reply:
haha, nah this is mainly just personal preference, I'm not a big fan of `f err := .. ; err ...` but tolerate it if the next line just returns the error, here since we did a bit more with the error I changed it. Happy to move it back though

Reply:
i don't mind either way!